slides/01-introduction.qmd

```diff
@@ -143,6 +143,54 @@ Many thanks to Julie Jung, Alison Hill, and Desirée De Leon for their role in c
     - Tuning hyperparameters
     - Wrapping up!
 
+## Today's data: frog hatching
+
+::: columns
+::: {.column width="60%"}
+```{r}
+#| echo: false
+#| out-width: '50%'
+
+library(dplyr)
+
+data("tree_frogs", package = "stacks")
+tree_frogs <- tree_frogs %>%
+  mutate(t_o_d = factor(t_o_d)) %>%
+  filter(!is.na(latency)) %>%
+  select(-c(clutch, hatched))
+
+tree_frogs %>%
+  head(10) %>%
+  knitr::kable() %>%
+  kableExtra::kable_styling(font_size = 30)
```

Comment:
I personally prefer the output you get from just printing the tibble out (i.e. no kable), but dont feel too strongly about that

Reply:
I think you're right!